testdata/memcached-molecule-operator/PROJECT

```diff
@@ -4,7 +4,7 @@
 # More info: https://book.kubebuilder.io/reference/project-config.html
 domain: example.com
 layout:
-- go.kubebuilder.io/v1
+- base.ansible.sdk.operatorframework.io/v1
```

Comment:
The term "base" currently refers to the default plugin configuration that doesn't use kustomize.v2 which generates the config.dir.

So we might want to change the registration line:
`plugin.WithName(ansible.Plugin{}.Name()),`
to use a more explicit name for the bundle. Instead of the current form:

`base.ansible.sdk.operatorframework.io/v1` we might wnat to align with something like
`.ansible.operatorframework.io/v1`